adminSiteClient/SuggestedChartRevisionApproverPage.tsx

```diff
@@ -348,7 +350,7 @@ export class SuggestedChartRevisionApproverPage extends React.Component<{
                 noSidebar
             >
                 <main className="SuggestedChartRevisionApproverPage">
-                    <h3>
+                    <h1>
```

Comment:
All these heading level changes seem a bit odd to me, they look a little too prominent now.
Not sure if these were all intentional?

Reply:
I changed them a bit, I think it looks fine.

I left to H1, the main title, and the one for the actual revision